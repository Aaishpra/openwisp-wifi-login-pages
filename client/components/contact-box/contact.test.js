import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import getConfig from "../../utils/get-config";
import Contact from "./contact";

const defaultConfig = getConfig("default");
const createTestProps = props => {
  return {
    language: "en",
    orgSlug: "default",
    contactPage: defaultConfig.components.contact_page,
    ...props,
  };
};

describe("<Status /> rendering", () => {
  let props;
  it("should render correctly", () => {
    props = createTestProps();
    const renderer = new ShallowRenderer();
    const component = renderer.render(<Contact {...props} />);
    expect(component).toMatchSnapshot();
  });
});
