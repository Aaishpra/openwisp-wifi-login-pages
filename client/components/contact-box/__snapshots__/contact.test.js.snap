// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Status /> rendering should render correctly 1`] = `
<React.Fragment>
  <div
    className="owisp-contact-container"
  >
    <div
      className="owisp-contact-inner"
    >
      <div
        className="owisp-contact-row"
      >
        <div
          className="owisp-contact-label"
        >
          E-mail
          :
        </div>
        <a
          className="owisp-contact-text"
          href="mailto:support@openwisp.co"
        >
          support@openwisp.co
        </a>
      </div>
      <div
        className="owisp-contact-row"
      >
        <div
          className="owisp-contact-label"
        >
          Helpdesk
          :
        </div>
        <a
          className="owisp-contact-text"
          href="tel:+789 948 564"
        >
          +789 948 564
        </a>
      </div>
      <div
        className="owisp-contact-links"
      >
        <a
          className="owisp-contact-twitter-link owisp-contact-link"
          href="https://twitter.com/openwisp"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="twitter"
            className="owisp-contact-twitter-image owisp-contact-image"
            src="/assets/default/twitter.svg"
          />
        </a>
        <a
          className="owisp-contact-facebook-link owisp-contact-link"
          href="https://facebook.com/openwisp"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="facebook"
            className="owisp-contact-facebook-image owisp-contact-image"
            src="/assets/default/facebook.svg"
          />
        </a>
      </div>
      <div
        className="owisp-contact-inner"
      />
    </div>
  </div>
</React.Fragment>
`;
